tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -267,6 +278,7 @@ def __init__(self, test_ctx: TestContext, *args, **kwargs):
             config_profile['machine_type']]
 
         tier_product = self.redpanda.get_product()
+        assert tier_product, "Could not get product into "
```

Comment:
`"Could not get product info"`

Reply:
Fixed in a subsequent big change to this same file, hope that's alright.

Reply:
Did you push this? Might be blind but can't see it.